.changeset/silver-eagles-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Permission the catalog-import route
```

Comment:
Please update this changeset with a diff that describes the changes required for an existing app that want to stay up to date.

Reply:
Done. Should this be a minor bump instead? not sure what our standard is around frontend changes.